torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1396,6 +1464,10 @@ RegisterOperators reg2({
 
     Operator("aten::remove(Tensor[](a!) self, Tensor el) -> ()",
         listRemove<Shared<TensorList>, at::Tensor>),
+    Operator("aten::index(Tensor[] self, Tensor el) -> int",
```

Comment:
We already have an `aten::index` which performs advanced indexing on tensors. Would it be a problem having conflicting names here?

Reply:
The jit tests ran successfully on my machine. However, it doesn't seem like a good idea to have two completely different operations share the same name.

Reply:
It should be fine, we have overload resolution logic that will select the correct version.